tests/ui/invalid_pyclass_args.stderr

```diff
@@ -0,0 +1,23 @@
+error: Expected one of freelist/name/extends/module, but got extend
+ --> $DIR/invalid_pyclass_args.rs:3:11
+  |
+3 | #[pyclass(extend=pyo3::types::PyDict)]
+  |           ^^^^^^
+
+error: Expected type path (e.g., my_mod::MyClass), but got "PyDict"
+ --> $DIR/invalid_pyclass_args.rs:6:21
+  |
+6 | #[pyclass(extends = "PyDict")]
+  |                     ^^^^^^^^
+
+error: Expected type name (e.g., Name), but got m :: MyClass
+ --> $DIR/invalid_pyclass_args.rs:9:18
+  |
+9 | #[pyclass(name = m::MyClass)]
+  |                  ^^^^^^^^^^
```

Comment:
I'm trying to decide if the `but got x` portion of the error messages is necessary.

On the plus side, they make the error description slightly nicer to read.

However, the invalid "but got" input is already highlighted in the source code beneath the error, so we're kinda repeating the information twice.

Also, in this `m::MyClass` case the formatting has been changed versus the original source code, which makes it a bit weird to read.

Reply:
cc @gilescope who reported the original issue.

Reply:
Thank you, I chose a simplfied one.